gapic/ads-templates/%namespace/%name/%version/__init__.py.j2

```diff
@@ -6,8 +6,8 @@ import importlib
 import sys
 
 
-if sys.version_info < (3, 7):
-    raise ImportError('This module requires Python 3.7 or later.')
+if sys.version_info < (3, 8):
+    raise ImportError('This module requires Python 3.8 or later.')
```

Comment:
It might be more user-friendly to raise a warning instead of an error here.

Reply:
In setup.py, we require python 3.8 so most folks on python 3.7 won't be able to install the library.

Reply:
We don't use the setup.py from the generator, but we also plan to set the minimum to 3.8 in our library, so 3.7 users will probably be blocked on install as well.